ex03.ex

```diff
@@ -73,15 +91,27 @@ defmodule Ex03 do
       iex> Ex03.list_contains([ 1, 2, 3, 4], 3)
       true
 
-      iex> Ex03.list_contains([ 1, 2, 3, 4], 3)
-      true
+      iex> Ex03.list_contains([ 2.25, 3.2, 1,2 ], 3)
+      false
 
-      iex> Ex03.list_contains([ 1, 2, 3, 4], 3)
+      iex> Ex03.list_contains([ 1, 3.5, 2.24, 3 ], 3)
       true
 
+      iex> Ex03.list_contains([ ], 4)
+      false
+
+      iex> Ex03.list_contains([ ], nil)
+      false
+
   """
 
-  def list_contains . .. "your code"
+  def list_contains([], _n), do: false
```

Comment:
# -1

This works, but I don't like the `cond` here. I think it's better written as

~~~elixir
  def list_contains([], _value),             do: false
  def list_contains([value | _rest], value), do: true
  def list_contains([_ | rest], value),      do: list_contains(rest, value)
~~~